docs/README.ja.md

```diff
@@ -1 +1,102 @@
-TODO
\ No newline at end of file
+<p align="center"><img src="/src/icons/icon@128.png" alt="holo-schedule logo"></p>
+
+# holo-schedule
+
+![Test status](https://github.com/YunzheZJU/holo-schedule/workflows/Test/badge.svg)
+
+—— Keep in touch with Hololive.
+
+ホロライブの配信を表示するブラウザ拡張です。
+
+[Eng](/README.md) [中文](docs/README.zh_CN.md)
+
+### 目次
+
+1. [インストール](#インストール)
+1. [対応ブラウザ](#対応ブラウザ)
+1. [依存関係](#依存関係)
+1. [ビルド](#ビルド)
+1. [テスト](#テスト)
+1. [開発](#開発)
+
+## インストール
+
+以下の公式ページでダウンロードできます。
+[Mozilla AMO](https://addons.mozilla.org/firefox/addon/holo-schedule/) 
+または  
+[Chrome Web Store](https://chrome.google.com/webstore/detail/holoschedule/fjicegllhddldnnkgfefblholeegpcad)
+
+[![Mozilla AMO](./get-the-add-on.png)](https://addons.mozilla.org/firefox/addon/holo-schedule/)
+[![Chrome Web Store](./available-in-the-chrome-web-store.png)](https://chrome.google.com/webstore/detail/holoschedule/fjicegllhddldnnkgfefblholeegpcad)
```

Comment:
I have created a [PR](https://github.com/YunzheZJU/holo-schedule/pull/41/files) to change all relative paths in README.md to absolute ones. 
```suggestion
[![Mozilla AMO](/docs/get-the-add-on.png)](https://addons.mozilla.org/firefox/addon/holo-schedule/)
[![Chrome Web Store](/docs/available-in-the-chrome-web-store.png)](https://chrome.google.com/webstore/detail/holoschedule/fjicegllhddldnnkgfefblholeegpcad)
```